mappings/net/minecraft/entity/EntityAttachments.mapping

```diff
@@ -1,9 +1,8 @@
 CLASS net/minecraft/class_9066 net/minecraft/entity/EntityAttachments
-	FIELD field_47751 DEFAULT Lnet/minecraft/class_9066;
```

Comment:
I couldn't find this field in the class. What's this? A ghost? 👻

Reply:
Hmm, maybe it was removed in the b snapshot? Most of my mapping was done on a

Reply:
Yeah I think it was removed in the b snapshot, and only existed in a.